src/Three.Legacy.js

```diff
@@ -1888,14 +1888,6 @@ export function CanvasRenderer() {
 
 //
 
-export function JSONLoader() {
```

Comment:
It's necessary to remove this entry otherwise `'JSONLoader' in THREE` always evaluates to `true`. That's a bit unfortunate for users who just want to create an instance of `JSONLoader`. They will get the message `THREE.JSONLoader is not a constructor`.

Reply:
We can keep that one and call the "new" one `THREE.LegacyJSONLoader`?

Reply:
Okay, renamed the loader from `THREE.JSONLoader` to `THREE.LegacyJSONLoader`.

Reply:
Phew, it's really not easy to get rid of the old stuff...